README.md

```diff
@@ -11,202 +11,24 @@ _Use your AsyncAPI definition to generate java code to subscribe and publish mes
-        cleanSession: false
-        keepAlive: 0
-        lastWill:
-          topic: /will
-          qos: 0
-          message: Guest gone offline.
-          retain: false
-    variables:
-      port:
-        enum:
-          - '8883'
-          - '8884'
-        default: '8883'
-    
-          
-defaultContentType: application/json
-
-channels:
-  smartylighting/streetlights/1/0/event/{streetlightId}/lighting/measured:
-    description: The topic on which measured values may be produced and consumed.
-    parameters:
-      streetlightId:
-        $ref: '#/components/parameters/streetlightId'
-    publish:
-      summary: Inform about environmental lighting conditions of a particular streetlight.
-      operationId: receiveLightMeasurement
-      message:
-        $ref: '#/components/messages/lightMeasured'
-
-  smartylighting/streetlights/1/0/action/{streetlightId}/turn/on:
-    parameters:
-      streetlightId:
-        $ref: '#/components/parameters/streetlightId'
-    subscribe:
-      bindings:
-        mqtt:
-          qos: 0
-          retain: false
-      operationId: turnOn
-      message:
-        $ref: '#/components/messages/turnOnOff'
-
-components:
-  messages:
-    lightMeasured:
-      name: lightMeasured
-      title: Light measured
-      summary: Inform about environmental lighting conditions of a particular streetlight.
-      payload:
-        $ref: "#/components/schemas/lightMeasuredPayload"
-    turnOnOff:
-      name: turnOnOff
-      title: Turn on/off
-      summary: Command a particular streetlight to turn the lights on or off.
-      payload:
-        $ref: "#/components/schemas/turnOnOffPayload"
-
-  schemas:
-    lightMeasuredPayload:
-      type: object
-      properties:
-        lumens:
-          type: integer
-          minimum: 0
-          description: Light intensity measured in lumens.
-          x-pi: false
-        sentAt:
-          $ref: "#/components/schemas/sentAt"
-    turnOnOffPayload:
-      type: object
-      properties:
-        command:
-          type: string
-          enum:
-            - on
-            - off
-          description: Whether to turn on or off the light.
-          x-pi: false
-        sentAt:
-          $ref: "#/components/schemas/sentAt"
-    sentAt:
-      type: string
-      format: date-time
-      description: Date and time when the message was sent.
-
-  parameters:
-    streetlightId:
-      description: The ID of the streetlight.
-      schema:
-        type: string
-```
 
-</p>
-</details>
+- Complete example for Kafka is [here](tests/kafka.yml). Notice information about binding.
+  ```yml
+  channels:
+    event.lighting.measured:
+      publish:
+        bindings:
+          kafka:
+            groupId: my-group
+        message:
+          $ref: '#/components/messages/lightMeasured'
+      subscribe:
+        message:
+          $ref: '#/components/messages/lightMeasured'
+  ```  
+- Complete example for MQTT is [here](tests/mqtt.yml).
```

Comment:
Can't we just link to the MQTT example in the `spec` repo?

Reply:
no, like in the case of Kafka example, Semen added some custom attributes to the examples, thus for the time being we need to have custom copies. Long term could be maybe one of first contributions for @iamVP7 to figure out, maybe we can just extend official examples with bindings added by Semen and do what you are suggesting. Something to investigate later I think @fmvilas

Reply:
Yeah, something to figure out later 👍

Reply:
@fmvilas so please approve 😄